src/javascript/crypto/e2e/openpgp/pgpmime/utils.js

```diff
@@ -120,7 +141,7 @@ e2e.openpgp.pgpmime.Utils.parseHeaderValue = function(text) {
     params[paramName] = goog.string.stripQuotes(paramVal, '"');
   });
 
-  return /**@type{e2e.openpgp.pgpmime.types.HeaderValue}*/ (
+  return /**@type{e2e.openpgp.pgpmime.types.HeaderValueWithParams}*/ (
```

Comment:
Same here. You can split the type declaration into a new line, or make it a full docblock:
```
/**
 * @type {}
 */
var x = ({})
return x;